### PR TITLE
Add University of Houston (UH) baseline design for luminosity detector

### PR DESCRIPTION
**Pull request by dhevang**
### Briefly, what does this PR introduce?
Add a baseline luminosity detector design (UHdesign) in the far-bwd region.
Components added:
- exit window
- spectrometer magnet
- four spectrometer trackers
- two spectrometer CALs

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?

Parts of the code used by the taggers have been modified as well (vacuum.xml changed to containmentBoxes.xml)

### CODE REVIEW

**Comment by wdconinc** [comment]
This will have to wait until after https://github.com/eic/epic/pull/272, and then it will have to go into the epic repository.

File: CMakeLists.txt [hunk 1]
```diff
@@ -67,7 +67,7 @@ set(a_lib_name ${PROJECT_NAME})
 
 dd4hep_configure_output()
 
-file(GLOB sources CONFIGURE_DEPENDS src/*.cpp)
+file(GLOB_RECURSE sources CONFIGURE_DEPENDS src/*.cpp ip6/far_backward/*.cpp)
```
Comment (wdconinc) [inline CMakeLists.txt:70]: Source files have to go in `src`

File: ip6/far_backward.xml [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
   <include ref="far_backward/magnets.xml"/>
-  <include ref="far_backward/vacuum.xml"/>
-  <include ref="far_backward/taggers.xml"/>
-  <!-- include ref="far_backward/lumi.xml"/ -->
   <include ref="far_backward/beamline_hadron.xml"/>
+  <include ref="far_backward/containmentBoxes.xml"/>
+  <include ref="far_backward/taggers.xml"/>
+  
+  <!-- Luminosity monitor -->
+ <include ref="far_backward/UH_LumiDesign.xml"/>
```
Comment (wdconinc) [inline ip6/far_backward.xml:11]: There are no 'ownership' signifiers in any of these systems. The design is not owned by the University of Houston, it's of the collaboration. The file names should not have these 'UH' prefixes and should in general follow the capitalization and style of the surrounding files.

File: ip6/far_backward/UH_LumiDesign.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd>
+
+   <include ref="UHdesign/exitWindow.xml"/>
+   <!-- <include ref="UHdesign/collimator.xml"/> -->
+   <!-- <include ref="UHdesign/sweepMagnet.xml"/> -->
+   <!-- <include ref="UHdesign/converter.xml"/> -->
```
Comment (wdconinc) [inline ip6/far_backward/UH_LumiDesign.xml:7]: No uncommented files since then they are not tested.

**Review by wdconinc — COMMENTED** [review]
Some comments for now, but this will need to go against epic anyway.

File: CMakeLists.txt (hunk 1 above)
Comment (dhevang) [reply CMakeLists.txt:70]: Ok, but can we at least make sub dirs inside the main src dir?  That way all the source code won't be dumped all in 1 place.

File: ip6/far_backward.xml (hunk 2 above)
Comment (dhevang) [reply ip6/far_backward.xml:11]: This was just a label in anticipation of other groups creating other Lumi designs.  But ok, I can call it "Design1" for instance.

**Pull request closed without merging**